python/ray/train/trainer.py

```diff
@@ -482,6 +483,25 @@ def latest_checkpoint(self) -> Optional[Dict]:
         """
         return self.checkpoint_manager.latest_checkpoint
 
+    @property
+    def best_checkpoint(self) -> Optional[Dict]:
+        """The best saved checkpoint.
+
+        Returns ``None`` if ``run()`` has not been called or if
+        ``train.save_checkpoint()`` has not been called from ``train_func``
+        within the most recent call to ``run``.
+        """
```

Comment:
Can you update the doc to match `best_checkpoint_path`? It may not be obvious what "best" means.

Reply:
Fixed!